### DISCUSSION TITLE
Problems configuring katex

### DISCUSSION THREAD

**Discussion by fcooper8472** [thuliteio/doks#1134] [Q&A]
I'm using @hyas/doks-core v1.1.1

Setting 

```
katex = true
```
in `params.toml` caused the site to fail to build: it seems as though katex isn't a dependency.

I worked around this by adding

```
"katex": "^0.16.9"
```
to `package.json`.

The next problem was that the katex css isn't imported.

I worked around this by adding

```
@import '../../../node_modules/katex/dist/katex.min';
```

to `_variables-custom.scss`, for lack of a better place to put it.

Both of these worked, and I now have maths rendered on my site exactly as I want it. But I assume this isn't how it's supposed to work! Is there a better way of configuring katex on a doks site?

Thanks!

**Comment by h-enk** [comment]
Actually, I dropped KaTeX support (mainly for performance reasons). But, I just released [@hyas/doks-core v1.1.2](https://github.com/gethyas/doks-core/releases/tag/v1.1.2)

Documentation will be available soon, but [Joe's examples](https://www.veriphor.com/articles/mathematical-expressions/) should help you get started for now.

I know some heavy users prefer MathJax — see e.g. [Neutrino Flavor Oscillations in Dense Media](https://neutrino.leima.is/) and [pntfx - Building Bridges between Mathematical Concepts](https://pontifex.hoou.tuhh.de/) — maybe their repositories could be of any help if you'd like to go that way.

**Reply by h-enk** [reply]
I guess I still could write a KaTeX support guide though — will do that

**Comment by fcooper8472** [comment]
Thanks @h-enk - some kind of maths support is essential for my use-case, but some instructions for getting set up even if it's not ready out-of-the-box would be great.

Thanks for your great work on doks!
